app/src/components/Form/Form.js

```diff
@@ -25,9 +40,18 @@ class Form extends React.Component {
       fieldIds.map(id => [id, fields[id].validation])
     );
     const validationSchema = Yup.object(validations);
+    console.log('validate', validate);
     return (
       <Formik
-        validate={validate || null}
+        validate={
```

Comment:
what's this section do?

Reply:
pull this out to a section into a separate method and call it remove visibility

Reply:
Now resolved! thanks!